README.rst

```diff
@@ -88,7 +88,7 @@ Then use it on your ``settings.py``.
    .. code-block:: python
 
      SECRET_KEY = config('SECRET_KEY')
-     DEBUG = config('DEBUG', default=False, cast=bool)
+     DEBUG = config('DEBUG', default=False, cast=strtobool)
```

Comment:
"Explicit is better than implicit."

it would be good to add `from distutils.util import strtobool` to example.